MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -90,7 +90,10 @@ open class BaseInstructionsBannerView: UIControl {
         }
     }
     
-    func set(_ instruction: VisualInstructionBanner?) {
+    /**
+     Updates the instructions banner info with a given `VisualInstructionBanner`.
+     */
+    public func updateInstruction(_ instruction: VisualInstructionBanner?) {
```

Comment:
@bsudekum what's the value of using a common denominator when the function only needs a specific info based on the signature?

Reply:
@bsudekum this will not be feasible for the way our current test cases have been set up. We need to create a dummy `routeProgress` for each one of them. This creates a dependency on `routeProgress`. Yes, it will make things easier for our devs for less verbose access of the currentVisualInstruction, however, it's only one level deep. Which isn't too complex (just one level deep).

https://github.com/mapbox/mapbox-navigation-ios/pull/1534/files#diff-11508d8606e1fc6044baab420712bc9cL83